appendix-sample-contract.md

```diff
@@ -37,21 +144,6 @@ to the content below and not to the personal identity of the author or to any
 identification or branding utilized by the author's law firm, Eckert Seamans.
 
 
-(TODO: Do we have better sample contract language on hand? I haven't come across 
```

Comment:
Just checking: are we able to remove this TODO because we've removed the material that would have been attributed to Foxman?

Reply:
No, that material is still in there. Our usage has been approved by Eckert Seaman, with the re-ordering that I just did (OSQA first, then their language). I out the TODO b/c it seems unnecessary at this point.